torch/distributed/_tensor/ops/math_ops.py

```diff
@@ -139,3 +140,35 @@ def var_correction_rule(op_schema: OpSchema) -> OutputSharding:
     return reduction_rule(
         op_schema, dims=dims, keep_dim=keep_dim, reduction_linear=False
     )
+
+
+@register_prop_rule("aten.native_layer_norm.default")
+def _prop_native_layer_norm(op_schema: OpSchema) -> OutputSharding:
```

Comment:
can we delete this? This is actually right now an experimental op in https://github.com/pytorch/tau/blob/main/spmd/tensor/experimental_ops.py#L10

Reply:
Sure, totally.